MovieApp/MovieApp/App/SceneDelegate.swift

```diff
@@ -1,8 +1,8 @@
 import UIKit
+import Resolver
```

Comment:
is this needed?

Reply:
Yes because I initialise AppModule with Resolver.main